### ISSUE TITLE
Update home page with link to validation tool

### ISSUE THREAD

**Issue by Melissa37** [JATS4R/elements#47]

**Comment by Klortho** [comment]
I am looking at how it might be possible to merge the validation tool, which is in the [validation repo, gh-pages branch](https://github.com/JATS4R/validation/tree/gh-pages) with the main site, which is in the [jats4r.github.io repo, master branch](https://github.com/JATS4R/jats4r.github.io). 

In fact, right now, things are scattered across *three* different branches, when you take the [validation repo, master branch](https://github.com/JATS4R/validation/tree/master) (which has the main sources for the validation schematrons) into account.

The easiest way to do this would be to combine the two repos, and move everything into the [jats4r.github.io repo, master branch](https://github.com/JATS4R/jats4r.github.io).  Does anyone have a problem if I do this?

There are two main reasons to do this:
* We can make the validator use the same page templates as the main site, so the validator pages will have the same look and feel.
* It makes development easier, since I (or anyone) can just clone one repo

The downside is that it will be harder to find the schematron files -- they'll be a little bit buried inside the website sources.

**Comment by Klortho** [comment]
I did what I described above:  I combined all of our stuff into the one repo/branch:  jats4r.github.io, master.  See [this pull request](https://github.com/JATS4R/jats4r.github.io/pull/3).

That means that:

1. The JATS4R/validation repo will go away:  I'll empty it out except for a README that says "deprecated", and gives instructions for where the files now reside.
2. The validator tool is now at [http://jats4r.github.io/validate/](http://jats4r.github.io/validate/). (I changed the name of the subdirectory from "validation")
